app/services/subscriptions/create_service.rb

```diff
@@ -58,7 +59,15 @@ def call
 
     private
 
-    attr_reader :customer, :plan, :params, :name, :subscription_at, :billing_time, :external_id, :current_subscription
+    attr_reader :customer,
+      :plan,
+      :params,
+      :name,
+      :subscription_at,
```

Comment:
Layout/ArgumentAlignment: Align the arguments of a method call if they span more than one line. (https://rubystyle.guide#no-double-indent)